util/limbo_utility.cpp

```diff
@@ -395,6 +395,9 @@ String LimboUtility::get_property_hint_text(PropertyHint p_hint) const {
 		case PROPERTY_HINT_ARRAY_TYPE: {
 			return "ARRAY_TYPE";
 		}
+		case PROPERTY_HINT_DICTIONARY_TYPE: {
```

Comment:
Should we use
```
#if GODOT_VERSION_AT_LEAST(4, 4)
... this ...
#endif // GODOT_VERSION_AT_LEAST
```
with

```
// util/limbo_compat.h
...
#if defined(LIMBOAI_GDEXTENSION)
    ...
   #include <godot_cpp/core/version.hpp>
#elif defined(LIMBOAI_MODULE)
    ...
    #include <core/version.h>
#endif

// Macro to check if the current Godot version is at least major.minor
#define GODOT_VERSION_AT_LEAST(major, minor) \
    (GODOT_VERSION_MAJOR > (major) || \
    (GODOT_VERSION_MAJOR == (major) && GODOT_VERSION_MINOR >= (minor)))
...
```

for backwards compatibility?

Alternatives: 
- Wait for 4.4 stable, drop support for 4.3

Reply:
> Wait for 4.4 stable, drop support for 4.3

That is the current process. Headers change location, API calls, function names, etc. This would bloat the codebase with preprocessor conditionals and increase maintenance (you would have to target 2-3 versions of Godot, and GDExtension!). So the process is to support the latest stable release, corresponding GDExtension, and switch gears to the next version somewhere around `beta3` or `rc1`.